### ISSUE TITLE
Documentation search is currently broken

### ISSUE THREAD

**Issue by sairuk** [mamedev/mame#12181]
### MAME version

0.263

### System information

docs.mamedev.org

Accessed through Firefox 120, although this is an upstream issue with the rtd project.

### Incorrect behaviour

Searching documentation on docs.mamedev.org does not complete instead reporting errors to the js console in developer tools

```
Uncaught ReferenceError: jQuery is not defined
    <anonymous> https://docs.mamedev.org/search.html?q=test&check_keywords=yes&area=default:129
search.html:129:7
```
```
Uncaught ReferenceError: jQuery is not defined
    <anonymous> https://docs.mamedev.org/search.html?q=test&check_keywords=yes&area=default:134
search.html:134:5
```

### Expected behaviour

Users are able to successfully search documentation

### Additional details

This appears to be an upstream issue with the sphinx_rtd_theme (half) removing jquery support https://github.com/readthedocs/sphinx_rtd_theme/issues/1546 other projects appear to have implemented workarounds, raising this for visibility on both projects.

**Comment by Firehawke** [comment]
Assigning this to myself and monitoring the RTD issue to see how they intend to handle this.

**Comment by Firehawke** [comment]
It doesn't look like RTD is going to fix their botched 2.0.0 release any time soon. There are a number of other smaller things I'd noticed that were concerning in 2.0.0, but the broken search was the primary problem. I've replaced the JQuery call on our local RTD version with a standard script tag, which is what I expect that the final upstream solution will be.

Search is functional again. I'm going to need to make a note to keep an eye out for future RTD/Sphinx updates to TRY to avoid breakages like this from happening again.
